google/cloud/bigquery/client.py

```diff
@@ -605,6 +606,70 @@ def get_dataset(self, dataset_ref, retry=DEFAULT_RETRY, timeout=None):
         )
         return Dataset.from_api_repr(api_response)
 
+    def get_iam_policy(
+        self, table, requested_policy_version=1, retry=DEFAULT_RETRY, timeout=None,
+    ):
+        if not isinstance(table, (Table, TableReference)):
+            raise TypeError("table must be a Table or TableReference")
+
+        if requested_policy_version != 1:
+            raise ValueError("only IAM policy version 1 is supported")
+
+        options = {}
+        body = {}
+        options["requestedPolicyVersion"] = requested_policy_version
+        body["options"] = options
```

Comment:
Simpler / clearer as a pure dict literal., e.g.:
```suggestion
        body = {"options": {"requestedPolicyVersion": 1}}
```